solutions/silver/usaco-896.mdx

```diff
@@ -7,6 +7,36 @@ author: Jesse Choe, Sofia Yang, Juheon Rhee
 
 [Official Analysis (C++)](http://usaco.org/current/data/sol_mountains_silver_jan19.html)
 
+## Explanation
+
+Notice that a mountain is obscured by another mountain if and only if the base or the mountain (the area the mountain takes up on the ground) is contained within the base of the other mountain.
+
+Since each mountain is a right triangle, for any mountain $\mathcal{i}$ the right side of the base is $\mathcal{i}_{x}+{i}_y$ and the left side is $\mathcal{i}_{x}-{i}_y$. This is shown below:
+
+![](usaco-896/mountains0.png)
+
+Therefore, for a mountain $\mathcal{i}$ to be contained in another mountain $\mathcal{j}$, $\mathcal{i}_{x}+{i}_{y}\le{j}_{x}+{j}_{y}$ and $\mathcal{i}_{x}-{i}_{y}\ge{j}_{x}-{j}_{y}$.
+
+We can see this in the graphs below:
+
+![](usaco-896/mountains1.png)
+
+In this case, In this case, $\mathcal{j}$, $\mathcal{i}_{x}+{i}_{y}\le{j}_{x}+{j}_{y}$ and $\mathcal{i}_{x}-{i}_{y}\ge{j}_{x}-{j}_{y}$, meaning that the left and right ends of the mountain $\mathcal{i}$ are contained within mountain $\mathcal{j}$.
+
+![](usaco-896/mountains2.png)
+
+In *this* case, $\mathcal{j}$, $\mathcal{i}_{x}+{i}_{y}\le{j}_{x}+{j}_{y}$, but $\mathcal{i}_{x}-{i}_{y}<{j}_{x}-{j}_{y}$, meaning that the right end of the mountain $\mathcal{i}$ is to the left of the rightmost end of mountain $\mathcal{j}$, but the left end of mountain $\mathcal{i}$ is not to the right of the left end of mountain $\mathcal{j}$, meaning it is not contained within mountain j.
+
+This same reasoning can also work if the mountains overlap, as shown below:
+
+![](usaco-896/mountains3.png)
```

Comment:
ok i'm all for handholding but this seems a bit _too_ obvious

i don't think it should take this many lines to say "a mountain is hidden if it's 100% in the interval of another"

Reply:
^^ this editorial is a tad bit too long and might confuse the reader more.